### PR TITLE
docs: perfect get_started.md of docker

### PR DESCRIPTION
**Pull request by thomasyxy**

### CODE REVIEW

File: docs/tutorials/get-started.md [hunk 1]
```diff
@@ -29,4 +29,32 @@ At this point, all the relevant environments required by Pipcook have been insta
 
 ```sh
 $ pipcook run examples/pipelines/mnist-image-classification.json
-```
\ No newline at end of file
+```
+
+### install docker image
```
Comment (yorkie) [inline docs/tutorials/get-started.md:34]:
```suggestion
### Install via Docker
```

File: docs/tutorials/get-started.md [hunk 2]
```diff
@@ -29,4 +29,32 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+
+First instal [Dockerfile](https://github.com/alibaba/pipcook/blob/master/Dockerfile) to local.
```
Comment (yorkie) [inline docs/tutorials/get-started.md:36]:
```suggestion
First install [Dockerfile](https://github.com/alibaba/pipcook/blob/master/Dockerfile).
```

File: docs/tutorials/get-started.md [hunk 3]
```diff
@@ -29,4 +29,32 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+
+open Dockerfile's path and install image.
```
Comment (yorkie) [inline docs/tutorials/get-started.md:38]:
```suggestion
Open Dockerfile's path and install image.
```

File: docs/tutorials/get-started.md [hunk 4]
```diff
@@ -29,4 +29,32 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+
+```sh
+$ git clone https://github.com/alibaba/pipcook.git && cd pipcook
+$ docker build -t taobao/pipcook .
```
Comment (yorkie) [inline docs/tutorials/get-started.md:42]:
```suggestion
$ docker build -t alibaba/pipcook .
```

File: docs/tutorials/get-started.md [hunk 5]
```diff
@@ -29,4 +29,32 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+```
+
+Check the image after installed.
```
Comment (yorkie) [inline docs/tutorials/get-started.md:45]:
```suggestion
Check if the image is installed successfully.
```

File: docs/tutorials/get-started.md [hunk 6]
```diff
@@ -29,4 +29,32 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+
+```sh
+$ docker images
+```
+
+| REPOSITORY | TAG | IMAGE ID | CREATED | SIZE |
+| --- | --- | --- | --- | --- |
+| taobao/pipcook | latest | c297c73d62d4 | 6 hours ago | 3.67GB |
```
Comment (yorkie) [inline docs/tutorials/get-started.md:53]: It's better to print the output from docker command, and wrap the above in code block.

File: docs/tutorials/get-started.md [hunk 7]
```diff
@@ -29,4 +29,32 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+
+Create container and run it.
```
Comment (yorkie) [inline docs/tutorials/get-started.md:55]:
```suggestion
And run in the following command:
```

File: docs/zh-cn/tutorials/get-started.md [hunk 8]
```diff
@@ -33,4 +33,32 @@ $ pipcook init
 
 ```sh
 $ pipcook run examples/pipelines/mnist-image-classification.json
-```
\ No newline at end of file
+```
+
+### 安装 docker 镜像
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started.md:38]:
```suggestion
### 通过 Docker 镜像安装
```

File: docs/zh-cn/tutorials/get-started.md [hunk 9]
```diff
@@ -33,4 +33,32 @@ $ pipcook init
...
+
+需要先下载[Dockerfile](https://github.com/alibaba/pipcook/blob/master/Dockerfile)到本地
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started.md:40]:
```suggestion
首先下载 [Dockerfile](https://github.com/alibaba/pipcook/blob/master/Dockerfile)。
```

File: docs/zh-cn/tutorials/get-started.md [hunk 10]
```diff
@@ -33,4 +33,32 @@ $ pipcook init
...
+
+打开Dockerfile所在目录，安装镜像
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started.md:42]:
```suggestion
打开 Dockerfile 所在目录，并构建镜像
```

File: docs/zh-cn/tutorials/get-started.md [hunk 11]
```diff
@@ -33,4 +33,32 @@ $ pipcook init
...
+
+```sh
+$ git clone https://github.com/alibaba/pipcook.git && cd pipcook
+$ docker build -t taobao/pipcook .
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started.md:46]: Ditto

File: docs/zh-cn/tutorials/get-started.md [hunk 12]
```diff
@@ -33,4 +33,32 @@ $ pipcook init
...
+```
+
+安装完成后查看
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started.md:49]:
```suggestion
安装完成后查看：
```

File: docs/zh-cn/tutorials/get-started.md [hunk 13]
```diff
@@ -33,4 +33,32 @@ $ pipcook init
...
+
+```sh
+$ docker images
+```
+
+| REPOSITORY | TAG | IMAGE ID | CREATED | SIZE |
+| --- | --- | --- | --- | --- |
+| taobao/pipcook | latest | c297c73d62d4 | 6 hours ago | 3.67GB |
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started.md:57]: Ditto

**Comment by thomasyxy** [comment]
> @thomasyxy Thanks for this improvements, I have some comments for this, and may I ask you sign our CLA as well?

Yes, of course~

**Comment by yorkie** [comment]
@thomasyxy There are some requested changes not updating, could you please fix them by commits?

File: docs/tutorials/get-started.md (hunk 6 above)
Comment (thomasyxy) [reply docs/tutorials/get-started.md]: Done.

File: docs/zh-cn/tutorials/get-started.md (hunk 13 above)
Comment (thomasyxy) [reply docs/zh-cn/tutorials/get-started.md]: Done.

File: docs/zh-cn/tutorials/get-started.md (hunk 11 above)
Comment (thomasyxy) [reply docs/zh-cn/tutorials/get-started.md]: Done.

**Comment by thomasyxy** [comment]
> @thomasyxy There are some requested changes not updating, could you please fix them by commits?

All changes has been completed.

File: docs/tutorials/get-started.md [hunk 14]
```diff
@@ -29,4 +29,30 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+```sh
+$ git clone https://github.com/alibaba/pipcook.git && cd pipcook
+$ docker build -t alibaba/pipcook .
+```
+
+Check if the image is installed successfully.
+
+```sh
+$ docker images
+```
```
Comment (yorkie) [inline docs/tutorials/get-started.md:49]:
```suggestion
REPOSITORY                                    TAG                 IMAGE ID            CREATED             SIZE
alibaba/pipcook                               latest              c297c73d62d4        7 hours ago         3.67GB
```
```

File: docs/tutorials/get-started.md [hunk 15]
```diff
@@ -29,4 +29,30 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+
+REPOSITORY                                    TAG                 IMAGE ID            CREATED             SIZE
+alibaba/pipcook                               latest              c297c73d62d4        7 hours ago         3.67GB
```
Comment (yorkie) [inline docs/tutorials/get-started.md:52]: And remove the above 2 lines.

File: docs/zh-cn/tutorials/get-started.md [hunk 16]
```diff
@@ -33,4 +33,30 @@ $ pipcook init
...
+```sh
+$ git clone https://github.com/alibaba/pipcook.git && cd pipcook
+$ docker build -t alibaba/pipcook .
+```
+
+安装完成后查看：
+
+```sh
+$ docker images
+```
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started.md:53]: See my suggestion on English version.

**Comment by yorkie** [comment]
@thomasyxy Thank you, LGTM but some small nits.

File: docs/tutorials/get-started.md [hunk 17]
```diff
@@ -29,4 +29,31 @@ At this point, all the relevant environments required by Pipcook have been insta
...
+$ docker build -t alibaba/pipcook .
+```
+
+Check if the image is installed successfully.
+
+```sh
+$ docker images
+REPOSITORY                                    TAG                 IMAGE ID            CREATED             SIZE
+alibaba/pipcook                               latest              c297c73d62d4        7 hours ago         3.67GB
+
```
Comment (yorkie) [inline docs/tutorials/get-started.md:51]:
```suggestion
```
```

File: docs/tutorials/get-started.md (hunk 15 above)
Comment (yorkie) [reply docs/tutorials/get-started.md:53]: @thomasyxy The is needed to be removed.

File: docs/tutorials/get-started.md (hunk 17 above)
Comment (yorkie) [inline docs/tutorials/get-started.md:51]: The code block need to be closed here.

File: docs/zh-cn/tutorials/get-started.md [hunk 18]
```diff
@@ -33,4 +33,30 @@ $ pipcook init
...
+
+REPOSITORY                                    TAG                 IMAGE ID            CREATED             SIZE
+alibaba/pipcook                               latest              c297c73d62d4        7 hours ago         3.67GB
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started.md:56]: Ditto

File: docs/tutorials/get-started.md (hunk 15 above)
Comment (thomasyxy) [reply docs/tutorials/get-started.md:50]: OK, I've done it.

**Comment by yorkie** [comment]
@thomasyxy It seems that your commits is broken with GitHub Actions environment, may I ask you to squash them and force push again to try?

**Comment by thomasyxy** [comment]
> @thomasyxy It seems that your commits is broken with GitHub Actions environment, may I ask you to squash them and force push again to try?

OK, I will reopen a PR.

**Comment by thomasyxy** [comment]
> > @thomasyxy It seems that your commits is broken with GitHub Actions environment, may I ask you to squash them and force push again to try?
> 
> OK, I will reopen a PR.

Maybe there is no need to reopen... 
It's work successfully.

**Comment by yorkie** [comment]
Good job :)

**Comment by thomasyxy** [comment]
@yorkie Thanks for your help, my English is bad.

**Comment by utkobe** [comment]
LGTM

**Pull request merged**